lib/api/src/modules/addons.ts

```diff
@@ -44,13 +44,11 @@ export interface Addon {
   disabled?: boolean;
   hidden?: boolean;
 }
-export interface Collection {
-  [key: string]: Addon;
+export interface Collection<T = Addon> {
```

Comment:
Maybe to rename? Because Collection makes more think to an array/list than to a map/dictionary

Reply:
It's exported. If we want to change the name, I'd like to do that in a separate PR.